webui/react/src/pages/F_ExpList/glide-table/ColumnPickerMenu.tsx

```diff
@@ -54,8 +55,9 @@ const ColumnPickerTab: React.FC<ColumnTabProps> = ({
 }) => {
   const filteredColumns = useMemo(() => {
     const regex = new RegExp(searchString, 'i');
+    const locations = Array.isArray(tab) ? tab : [tab];
```

Comment:
nit sugg: I've seen this pattern a couple times before, maybe it could be extracted into a util function, something like `ensureArray`?

Reply:
sounds good.